runtime/jvmTest/src/kotlinx/serialization/SerializeFlatTest.kt

```diff
@@ -77,6 +78,10 @@ object CustomSerializer : KSerializer<Custom> {
             "value2" -> 1
             else -> -1
         }
+
+        override fun getElementAnnotations(index: Int): List<Annotation> = emptyList()
+        override fun getElementDescriptor(index: Int): SerialDescriptor = TODO()
```

Comment:
Since it is not used in this test, probably better call `assert(false, "should not be called")`